library/api_models.py

```diff
@@ -0,0 +1,197 @@
+from datetime import datetime
+from uuid import UUID
+from flask_restx import Api, fields
+from pydantic import AliasChoices, BaseModel, EmailStr, Field, ConfigDict
+
+import typing
+from typing import List, Optional, Union, TypeVar, Generic
+from dataclasses import dataclass, Field, fields as dataclassFields
+from flask_restx import fields as flaskRPFields
+
+
+
```

Comment:
Are these what you mean by strongly typed?

Reply:
Ah no, this should be deleted thanks. Restx is a library for APIs in flask but it's too weak and hard to get working (bad documentation)

Reply:
This is great actually!